app/routes/index.js

```diff
@@ -1,9 +1,25 @@
 var express = require('express')
 var router = express.Router()
 const documentationUrl = 'https://documenter.getpostman.com/view/1159934/SWDze1Rp'
+const Url = require('../models/ShortUrl');
 /* GET home page. */
 router.get('/', function (req, res, next) {
   res.redirect(documentationUrl)
 })
 
+router.get('/:shorturl',async (req,res)=>{
+  try {
+    const url = await Url.findOne({urlcode: req.params.shorturl});
+    
+    if (url) {
+      return res.redirect(url.longurl);
+    }
+    else{
+      return  res.json('No url found!');
+    }
+  } catch (error) {
+    res.json('Server error!')
+  }
+});
+
```

Comment:
We can have this in a separate file. `urlShortner.js`

Reply:
Done, I've added this in the url.js file in which there was POST Request only.

Reply:
But @devesh-verma this GET request should remain in the Index.js file, if I try to move it in any other file then instead of shortURL being `127.0.0.1:3000/1584954626498` it will become `127.0.0.1:3000/shortUrl/1584954626498` as the handler will change. Or we can add a redirect from the Index file. 
![image](https://user-images.githubusercontent.com/48180687/77305459-08a89800-6d1c-11ea-935f-3c2eab668c69.png) 

Please let me know using which approach I should move forward?

Reply:
this is fine, nothing wrong here.